internal/v2/controller/http/controller.go

```diff
@@ -0,0 +1,80 @@
+//
+// Copyright (c) 2020 Intel Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+//
+
+package http
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+	"time"
+
+	"github.com/edgexfoundry/go-mod-core-contracts/clients"
+	"github.com/edgexfoundry/go-mod-core-contracts/clients/logger"
+	commonV2 "github.com/edgexfoundry/go-mod-core-contracts/v2"
+	"github.com/edgexfoundry/go-mod-core-contracts/v2/dtos/common"
+	"github.com/google/uuid"
+)
+
+// V2Controller controller for V2 REST APIs
+type V2Controller struct {
+	lc logger.LoggingClient
+}
+
+// NewV2Controller creates and initializes an V2Controller
+func NewV2Controller(lc logger.LoggingClient) *V2Controller {
+	return &V2Controller{
+		lc: lc,
+	}
+}
+
+// NewPingResponse
+// TODO: move into go-mod-core-contracts
+func NewPingResponse() common.PingResponse {
```

Comment:
Added to core contracts in this PR:
https://github.com/edgexfoundry/go-mod-core-contracts/pull/260